packages/wdio-cucumber-framework/src/index.ts

```diff
@@ -213,21 +213,26 @@ class CucumberAdapter {
                 this._specs = this.filterSpecsByTagExpression([this._specs]).flat(1)
             }
 
-            const gherkinMessageStream = GherkinStreams.fromPaths(this._specs, {
-                defaultDialect: this._cucumberOpts.featureDefaultLanguage,
-                newId: this._newId
-            })
-
-            await parseGherkinMessageStream({
-                cwd: this._cwd,
-                eventBroadcaster: this._eventBroadcaster,
-                gherkinMessageStream,
-                eventDataCollector: this._eventDataCollector,
-                order: this._cucumberOpts.order,
-                pickleFilter: this._pickleFilter
-            })
+            if (this._specs.length){
```

Comment:
Let's return early
```suggestion
            if (this._specs.length === 0){
                this._hasTests = false
                return this
            }
```

can we move this and the if statement above outside of the try/catch?